libs/documentation/src/lib/storybook/tabs/tabs-custom-header/tabs-custom-header.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="tabs-wrapper">
+  <sds-tabs [(selectedTab)]="selectedTab">
+    <sds-tab-panel #tab1 [tabHeader]="tab1Header">
```

Comment:
Instead of using the icon in the custom header. Can you add test and icon with some CSS on it

Reply:
Like all three in the same header? Text, icon, and styling?

Reply:
Yes all three

Reply:
That can be done. My understanding was that for these types of demos we are trying to show what the component can do in the most basic way. With the idea being that a dev can come to the site needing a basic feature, like just an icon, and find an example that does just that. I'm concerned that combining all three into one might complicate the above scenario as that user would need to parse through what is and is not required to get just an icon.

If a dev is coming to the site to figure out how to do text, icon, and styling in one tab, they have a separate example of all three. I'd expect that they can figure out how to combine the three to get the result they are looking for.

Reply:
It's reverses the teams coming here and getting confused with icon tab is a customer header, same thing happened with me and Dave. If we add icon with some text at least it will be clear that it can be customizable other wise it looks like a input for the icon header tab.

Reply:
Again I don't understand how it is confusing.

Reply:
Additionally, we already have a demo in the current documentation that only icon headers:
<img width="1050" alt="Screen Shot 2023-02-22 at 8 47 06 AM" src="https://user-images.githubusercontent.com/72805180/220638371-235c5fc3-8751-4ba7-a785-67816aac92dd.png">

Reply:
By seeing the just demo it doesn't look like a custom header, it's more kind of an icon header

Reply:
Custom header implies that the header is not using the default template. This demo is not using the default template. A dev can replace the default header with any HTML elements. In the particular demo we are discussing I have demonstrated three possibilities, which can be easily combined if a dev needs any combination of them.

1) Just an icon
2) Unstyled HTML elements
3) HTML elements with a class provided to stylize the elements

I'm still not following how displaying one of three tabs as an icon would cause devs to think the tabs are not customizable.

If you still think we should do something different, lets discuss during parking lot, talking through it may help me to better understand your concern.